draft-packages/form/KaizenDraft/Form/Primitives/Label/Label.module.scss

```diff
@@ -38,15 +43,23 @@ $label-start-margin: 10px;
 // This is to override bootstrap styles. Remove when appropriate
 .text,
 :global(.ideal-sans) .text {
-  @include kz-typography-heading-6;
-  @include ca-inherit-baseline;
+  position: static;
```

Comment:
Do you definitely need the `position: static` on these? Would be good to not have that anymore

Reply:
`@include ca-inherit-baseline;` mixin is equal to `position: static;`. I will try removing and run chromatic on it

Reply:
No chromatic changes :)